bigbluebutton-html5/imports/ui/components/dropdown/list/title/component.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component, PropTypes } from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+import styles from '../styles';
+
+const intlMessages = defineMessages({
+  titleContextLabel: {
+    id: 'app.dropdown.list.titleContextLabel',
+    description: 'adds context to dropdown list title',
+  },
+});
+
+class DropdownListTitle extends Component {
+
+  render() {
+    const { label, intl } = this.props;

```

Comment:
Instead of a `prop.label` the label should be the component children and used as below:
`<DropdownListTitle>My Awesome Title</DropdownListTitle>`